packages/contracts-bedrock/snapshots/abi/CrossL2Inbox.json

```diff
@@ -243,4 +243,4 @@
     "name": "TargetCallFailed",
     "type": "error"
   }
-]
```

Comment:
Looks like contract-reviewers were pinged due to this whitespace change.  Looks unrelated - can this be cut?

Reply:
Technically it can be cut yes, but my IDE linter thought it was important enough that it should be included whenever I save the file. I can remove it via `vi` or something.

Reply:
```
truncate -s -1 packages/contracts-bedrock/snapshots/abi/CrossL2Inbox.json
```
👍

Reply:
I think this is what was causing the [bedrock checks to fail](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/60960/workflows/13394064-5c96-4d71-84c6-9a7c4baed581/jobs/2556155) previously.